src/rgw/const.py

```diff
@@ -16,6 +16,14 @@
 
 from enum import Enum
 
+
+# Filenames for support bundle
```

Comment:
\# RGW log files to collect in support bundle

Reply:
done